<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setting Color</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Color of living cells:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QPushButton" name="changeA">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBoxA">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>65</height>
             </size>
            </property>
            <property name="title">
             <string>Current color</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <widget class="QLabel" name="colorA">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="infoA">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Color of dead cells:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QPushButton" name="changeD">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBoxD">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>65</height>
             </size>
            </property>
            <property name="title">
             <string>Current color</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="1">
              <widget class="QLabel" name="colorD">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="infoD">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Color of cell borders:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="1">
           <widget class="QPushButton" name="changeB">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="gridBox">
            <property name="text">
             <string>Grid</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QGroupBox" name="groupBoxB">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>65</height>
             </size>
            </property>
            <property name="title">
             <string>Current color</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="infoB">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="colorB">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="applyButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="cancelButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
